Sources/MapboxDirections/DirectionsResult.swift

```diff
@@ -20,8 +20,7 @@ open class DirectionsResult: Codable {
     }
     
     // MARK: Creating a Directions Result
-    
-    init(legs: [RouteLeg], shape: LineString?, distance: CLLocationDistance, expectedTravelTime: TimeInterval) {
+    public init(legs: [RouteLeg], shape: LineString?, distance: CLLocationDistance, expectedTravelTime: TimeInterval) {
```

Comment:
Per the class documentation, it is intentional that clients can’t create DirectionsResult instances directly, because DirectionsResult is an abstract superclass of Route and Match. #406 incorrectly removed all the public initializers from both classes. Can you restore them to those classes instead? Thanks!

Reply:
Thanks for pointing that out. Added public init for `Route`, inits were available for `Match`, but public API docs had to be updated.